services/node_manager/src/main/java/com/futurewei/alcor/nodemanager/service/implement/NodeServiceImpl.java

```diff
@@ -100,30 +100,21 @@ private void handleCreateNodeBulkRequest(List<NodeInfo> nodeInfos) {
         }
     }
 
-    private boolean augmentNodeInfosWithNcmUri(List<NodeInfo> nodeInfos) {
+    private void augmentNodeInfosWithNcmUri(List<NodeInfo> nodeInfos) throws Exception {
         for (int i = 0; i < nodeInfos.size(); ++i) {
             NodeInfo ni = nodeInfos.get(i);
-            if (addNcmUriToNodeInfo(ni) == false)
-                return false;
+            addNcmUriToNodeInfo(ni);
```

Comment:
@pkommoju why you need to loop through every node here?

Reply:
@pkommoju I see, this is for node file upload.

Reply:
Every nodeinfo needs to have it's ncmid resolved to correct ncmuri.